.cfignore

```diff
@@ -2,5 +2,6 @@
 .hoodie
 .hoodierc
 *.log
+dist
```

Comment:
no wait, I think we need the `dist/` folder, it’s where the built assets are in for the app and visualization